apps/condo/domains/user/schema/AuthenticateUserWithEmailAndPasswordService.js

```diff
@@ -111,7 +111,14 @@ const AuthenticateUserWithEmailAndPasswordService = new GQLCustomSchema('Authent
                 }
 
                 const { keystone } = getSchemaCtx('User')
-                const token = await context.startAuthedSession({ item: user, list: keystone.lists['User'] })
+                const token = await context.startAuthedSession({
+                    item: user,
+                    list: keystone.lists['User'],
+                    meta: {
+                        source: 'gql',
```

Comment:
```suggestion
                        source: 'API',
```

Reply:
So, sessions created as a result of some gql operation is marked as `gql`-sourced